package.json

```diff
@@ -34,7 +34,7 @@
     "release": "run() { .scripts/release.sh $1 $2; }; run",
     "lint": "pnpm recursive exec --filter @cycle/* -- pnpm run lint",
     "test": "pnpm recursive exec --filter @cycle/* -- pnpm test ",
-    "test-ci": "pnpm recursive exec --filter @cycle/* -- pnpm run test-ci"
+    "test-ci": "pnpm recursive exec --workspace-concurrency 1 --filter @cycle/* -- pnpm run test-ci"
```

Comment:
@zkochan Thanks! But we didn't want the column separation output we got in the stdout, so we went for this instead. It would be nice if there was an option to disable that formatting.

Reply:
Yes! This would be nice, should I open an issue on the pnpm github?